.zenodo.json

```diff
@@ -13,11 +13,11 @@
     "contributors": [
         {
             "name": "Johann-Mattis List",
-            "type": "Other"
+            "type": "Editor"
         },
         {
             "name": "Mei-Shin Wu",
-            "type": "Other"
+            "type": "Editor"
```

Comment:
Mei-Shin is not an editor on this

Reply:
you are right, probably `Other` would be a better fit on this case. Can I have confirmation on this from you, @LinguList, please?

Reply:
Agreed with @FredericBlum